packages/keyring-controller/src/KeyringController.ts

```diff
@@ -1883,6 +1843,57 @@ export class KeyringController extends BaseController<
     this.#setUnlocked();
   }
 
+  /**
+   * Internal non-exclusive method to verify the seed phrase.
+   *
+   * @returns A promise resolving to the seed phrase as Uint8Array.
+   */
+  async #verifySeedPhrase(): Promise<Uint8Array> {
+    this.#assertControllerMutexIsLocked();
```

Comment:
This forces any caller to fist lock KeyringController mutex (e.g. with `#withControllerLock`)